sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -211,7 +218,7 @@ class MapboxMap internal constructor(
       nativeObserver.awaitingStyleGetters.clear()
     }
     onMapLoadErrorListener?.let {
-      addOnMapLoadErrorListener(it)
+      removeOnMapLoadErrorListener(it)
```

Comment:
Why change to remove here?

Reply:
@Chaoba 
we have adding map load error listener here
https://github.com/mapbox/mapbox-maps-android/blob/b9eaac796f1167c545e607dee427e952f57ab066/sdk/src/main/java/com/mapbox/maps/MapboxMap.kt#L180-L182
and then again after style is loaded
https://github.com/mapbox/mapbox-maps-android/blob/b9eaac796f1167c545e607dee427e952f57ab066/sdk/src/main/java/com/mapbox/maps/MapboxMap.kt#L213-L215
We need to remove that listener after map is loaded successfully.
Tagging @tobrun because he is reworking in general that approach, I just assume that this PR will get merged earlier and fix that typo.